src/menu.js

```diff
@@ -0,0 +1,41 @@
+const renderMenuPage = (function () {
```

Comment:
- [ ] Use arrow functions instead of the function keyword. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from Pathwright)